### PR TITLE
The syndicate are led by aliens.

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
Adds an alien-tech research disk to scientist/roboticist uplink for 12tc and makes alien technologies more consistent with illegal tech and itself. Also adds brainwashing and alien tech disks to roboticist uplink.

## Why It's Good For The Game
science traitors can now do alien gimmicks, and alien tech is centered around scaling into late-game. good use of this item requires knowing how to boost point production in science (toxins mainly) and what potential gimmicks you can do with techs that have alientech as a prerequisite.

## Changelog
:cl: tinfoil hat wearer
add: Added a new alien technology disk to scientist and roboticist uplinks that allows them to research the heavily-guarded secrets of the Grays.
tweak: Alientech is now the only Hidden alien research. To compensate for this, alien_bio and alien_engi have had their research costs doubled and now require advanced surgery tools and experimental tools respectively to research. Their export price is also halved.
balance: roboticists now have brainwashing disks AND alien technology added to their role-restricted uplink section. alien technology gives them brainwashing at a much later date, so brainwashing is the much cheaper option for instant power. makes logical sense because doctors get it as well because they do surgery, and roboticists can now either choose to brainwash people for less price but less power or emag borgs for higher prices, limited uses, but higher power.
/:cl:

### CODE REVIEW

File: code/modules/research/techweb/all_nodes.dm [hunk 1]
```diff
@@ -1027,26 +1027,19 @@
...
-	prereq_ids = list("alientech", "adv_engi")
+	prereq_ids = list("alientech", "exp_tools")
 	design_ids = list("alien_wrench", "alien_wirecutters", "alien_screwdriver", "alien_crowbar", "alien_welder", "alien_multitool")
-	boost_item_paths = list(/obj/item/screwdriver/abductor, /obj/item/wrench/abductor, /obj/item/crowbar/abductor, /obj/item/multitool/abductor, /obj/item/stock_parts/cell/infinite/abductor,
-	/obj/item/weldingtool/abductor, /obj/item/wirecutters/abductor, /obj/item/circuitboard/machine/abductor, /obj/item/abductor_baton, /obj/item/abductor)
-	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
-	export_price = 20000
-	hidden = TRUE
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 5000)
+	export_price = 10000
```
Comment (Trilbyspaceclone) [inline code/modules/research/techweb/all_nodes.dm:1042]: export price means nothing

Comment (ghost) [reply code/modules/research/techweb/all_nodes.dm:1042]: ok but if it does in the future then might as well i guess

Comment (Trilbyspaceclone) [reply code/modules/research/techweb/all_nodes.dm:1042]: Ive messed with the code for it and techwebs legit disables 100%

Comment (Trilbyspaceclone) [reply code/modules/research/techweb/all_nodes.dm:1042]: Its just line padding at this point and should not be a active var

**Comment by ghost** [comment]
what the fuck travis this compiled
i'm gonna stab you

**Comment by ghost** [comment]
travis can live for now

one more fuck-up and we're replacing you with ghommie

**Review by silicons — APPROVED** [review]

**Pull request merged**